bitnami/fluentd/Chart.yaml

```
@@ -25,4 +25,4 @@ name: fluentd
 sources:
   - https://github.com/bitnami/bitnami-docker-fluentd
   - https://www.fluentd.org/
-version: 4.1.7
+version: 4.1.8
```

As it is a feature addition instead of bug fixing, can you please bump the minor version instead of the patch one?
```suggestion
version: 4.2.0
```